### PR TITLE
fix: text does not overlap icon in alerts

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Chromatic

https://1040-va-alert-icons--60f9b557105290003b387cd5.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1040

## Testing done
Storybook

## Screenshots
Screen Shot 2022-07-27 at 09 49 20

## Acceptance criteria
- [x] Text in alerts does not overlap the close icon

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-alert/va-alert.css [hunk 1]
```diff
@@ -166,3 +166,7 @@ i {
 i.fa-times-circle::before {
   content: '\F057'; /* fa-times-circle*/
 }
+
+:host([closeable]:not([closeable='false'])) .alert {
+  padding-right: 5.7rem;
```
Comment (andresriveratoro) [inline packages/web-components/src/components/va-alert/va-alert.css:171]: This ends up being 16px away from the close icon.

Comment (bkjohnson) [reply packages/web-components/src/components/va-alert/va-alert.css:171]: @humancompanion-usds seemed [interested in having the padding be equal on both sides](https://github.com/department-of-veterans-affairs/component-library/pull/429#discussion_r875850031). Is that still something that we want?

Comment (k80bowman) [reply packages/web-components/src/components/va-alert/va-alert.css:171]: We can try it, but I worry that we're going to end up adding back the [problem we were trying to solve](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/658) with that earlier PR. If we add too much padding, alerts with a lot of text (which maybe shouldn't be a thing, but it is) get really long on smaller screens.

Comment (andresriveratoro) [reply packages/web-components/src/components/va-alert/va-alert.css:171]: This padding only takes effect if there's a close icon and it maintains the same padding on both sides relative to the icons. The `Dismissable Background Only Icon` left icon is 20px from the edge. I could update that icon to be 16px from the edge if we want.

**Review by bkjohnson — APPROVED** [review]

**Comment by andresriveratoro** [comment]
Update Sketch to use 16px margin between the close icon and the body container for designs that include the close icon.

**Review by k80bowman — APPROVED** [review]

**Pull request merged**